pyproject.toml

```diff
@@ -38,16 +38,17 @@ repository = "https://github.com/Sindri-Labs/sindri-python"
 version = "0.0.0"
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.8"
 requests = "^2.13.0"
 
 [tool.poetry.group.dev.dependencies]
+python = ">=3.8.1"
```

Comment:
I don't know about this dependency group. Is >=3.8.1 correct when we allow 3.8 minimum elsewhere?

Reply:
This allows us to do `poetry install --no-dev` to install just the core dependency group when running in python `3.8.0`. But if you want to install the `dev` dependency group, you'll need `3.8.1` because `flake8` requires `3.8.1`.